### PR TITLE
added new viewport for few tests

### LINKED ISSUE #4056: Write Cypress tests for the updated Front Page Top Story component
**Is your feature request related to a problem? Please describe.**
Once the updated Front Page Top Story component design has been merged, we must write Cypress tests for the Desktop variant. An annotated image of the redesign has been added below. 

We should check that the following is tested...

**Section No.1**
- Headline
- Summary
- Image
- Time/Datestamp

**Section No.2**
- Headline
- No Summary
- Image
- Time/Datestamp

**Section No.3**
- Headline
- No Summary
- No Image
- Time/Datestamp

**Describe the solution you'd like**
We need to implement Cypress tests that test for the presence of the components on the screen. Any granular testing such as styling, etc. should be done within the unit tests.

**Testing notes**
Making sure that the e2es pass on local, test & live.

image

This is currently blocked on this milestone - https://github.com/bbc/psammead/milestone/5
Once this has been finished the designs will then need to be implemented into Simorgh, which is when we can begin to write the Cypress tests.

### PR DESCRIPTION
**Pull request by PriyaKR**
Resolves #4056 
**Overall change:** 
_Modified existing frontpoage tests for new desktop layout >1008+._

**Code changes:**
- _By default cypress tests are run at a viewport width `1000x660` modified the frontpage tests for asserting there is atleast one section and promo(with h2,img,paragraph and timestamp) to test at `1008*768` viewport size ._
- _There is a `cy.viewport(1008x768)` command at the start of the test which runs once and the rest of the tests run at default viewport width.Could make it to run it before each test by changing `before` to `beforeEach` but the rest of the tests wouldnt vary depending upon the viewport so have modified these two tests_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: cypress/integration/pages/frontPage/tests.js [hunk 1]
```diff
@@ -123,7 +125,7 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
               .should('be.visible')
               .find('a')
               .should('have.attr', 'href');
-
+            // This check would only run for the top stories section and not anyother sections
```
Comment (DenisHdz) [inline cypress/integration/pages/frontPage/tests.js]:
```suggestion
            // This check would only run for the top stories section and not any other sections
```

**Review by DenisHdz — APPROVED** [review]

**Review by AlistairGempf — APPROVED** [review]
Looks good

File: cypress/integration/pages/frontPage/tests.js [hunk 2]
```diff
@@ -70,6 +70,7 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
 
       describe('Section', () => {
         it('should be labelled by a visible section label', () => {
```
Comment (sareh) [inline cypress/integration/pages/frontPage/tests.js]:
```suggestion
        it('should be labelled by a visible section label for 1008px & 320px layouts', () => {
```

File: cypress/integration/pages/frontPage/tests.js [hunk 3]
```diff
@@ -90,6 +91,7 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
         });
 
         it('should contain at least one story promo', () => {
```
Comment (sareh) [inline cypress/integration/pages/frontPage/tests.js]:
```suggestion
        it('should contain at least one story promo for 1008px & 320px layouts', () => {
```

**Review by sareh — COMMENTED** [review]
Just a couple of description suggestions to clarify we're testing the two different layouts/breakpoints.

**Review by sareh — APPROVED** [review]

**Review by sareh — APPROVED** [review]
👍 Thanks for the update, Priya.

**Pull request merged**